.gitignore

```diff
@@ -0,0 +1,4 @@
+# .gitignore
+#files here are ignored
```

Comment:
- [ ] OPTIONAL
Your image directory contains two large bin files as can be seen below. Kindly include these files in the gitignore file.
![image](https://user-images.githubusercontent.com/12745474/125702446-04941a4e-594f-4537-bb73-c489b8b95cc4.png)

![image](https://user-images.githubusercontent.com/12745474/125700224-f7b2a892-f0dd-4d46-85d5-b66d76d772c4.png)